### PR TITLE
Persona: mergeStyles part 3, split sub-components

### PR DESCRIPTION
**Pull request by jordandrako**
#### Pull request checklist

- [x] Include a change request file using `$ npm run change`

#### Description of changes

- Split PersonaCoin.tsx into PersonCoin.tsx and PersonaCoin.base.tsx
- Split PersonaPresence.tsx into PersonPresence.tsx and PersonaPresence.base.tsx
- Move PersonaCoin and PersonaPresence into subfolders.

### CODE REVIEW

**Comment by jordandrako** [comment]
@dzearing @oengusmacinog Should sub-components like PersonaCoin and PersonPresence live in their own folders like the Button folder structure to cut down on clutter?

File: packages/office-ui-fabric-react/src/components/Persona/index.ts [hunk 1]
```diff
@@ -2,4 +2,5 @@ export * from './Persona';
 export * from './Persona.base';
 export * from './Persona.types';
 export * from './PersonaCoin';
+export * from './PersonaCoin.base';
```
Comment (oengusmacinog) [inline packages/office-ui-fabric-react/src/components/Persona/index.ts]: probably want to add PersonaPresence.base too

Comment (jordandrako) [reply packages/office-ui-fabric-react/src/components/Persona/index.ts]: Left out for now as Persona.types export PersonaPresence enum so the names conflict. What are your thoughts on this:

```tsx
export { PersonaPresence as PersonaPresenceIcon } from './PersonaPresence';
export { PersonaPresenceBase as PersonaPresenceIconBase } from './PersonaPresence.base';
```

**Comment by oengusmacinog** [comment]
For File structure, I think what you have is good.  Might start getting confusing as we scale, but essentially nested components would be intended to be subcomponents of the folder they are nested in, even though each sub-component can be used individually if desired.  Still seems cleaner to structure this way, just should document some standards of when to nest and export a component

**Pull request merged**